HW_1/task1.py

```diff
@@ -0,0 +1,16 @@
+#Задача 2: Найдите сумму цифр трехзначного числа.
+
+#*Пример:*
+
+#123 -> 6 (1 + 2 + 3)
+#100 -> 1 (1 + 0 + 0) 
+
+#n = input("Введите трехзначное число: ")
+#n = int(n)
```

Comment:
Странно, только что проверял работу с точно такой же ошибкой.
```suggestion
#n = input("Введите трехзначное число: ")
```